src/dbnode/server/server.go

```diff
@@ -564,9 +565,27 @@ func Run(runOpts RunOptions) {
 
 	if cfg.DebugListenAddress != "" {
 		go func() {
-			if err := http.ListenAndServe(cfg.DebugListenAddress, nil); err != nil {
+			mux := http.DefaultServeMux
+			cpuProfileDuration := 1 * time.Second
```

Comment:
I'd probably do `5s`. Making it configurable would be nice if you have time, but not a huge deal since they can always do it manually if they want. Generally we make stuff like this a `const` at the top of the file though

Reply:
I set it to `5s` and as a `const`. Would like to keep this PR small, if possible, and make it configurable in a different PR.